### ISSUE TITLE
Not working on motorola one macro

### ISSUE THREAD

**Issue by castellvania93** [phhusson/treble_experimentations#1155]
Any clue, i've installed version 213 and my phone ended with a bootloop.

**Comment by phhusson** [comment]
Which kind of bootloop?
Please describe in more details

**Comment by realaltffour** [comment]
Hey, I have a Motorola One Macro.
I've installed the android 10 ab system image.
The phone booted.
Showed the bootloader unlocked warning message.
Showed the OS fingerprint (I don't know what this is, but usually if not empty device boots, this is from my experience with the device).
Then showed the normal motorola screen that came with the device.
Then the android logo appeared and that's where it just gets stuck here.
The adb logcat command just shows waiting for device. I guess this means that the device didn't start the adb deamon.

I want to clear some stuff too, 
So what is different with this project and the google developer official page? The readme only shows project operations

**Comment by phhusson** [comment]
This projects targets maximum compatibility with a maximum number of
devices.

I'm guessing `adb devices` will show you the device, but as unauthorized

**Comment by realaltffour** [comment]
@phhusson Can confirm it does show with no permissions

**Comment by phhusson** [comment]
If it says "no permissions" it means your adb isn't properly setup.
Do `pkill -9 adb; sudo adb devices -l` and it will no longer say "no
permissions"

**Comment by realaltffour** [comment]
Ok. Now `sudo adb devices` shows unauthorized. Running `sudo adb logcat` shows waiting for device. I've formatted the devices' userdata by the factory reset option at the bootloader.

**Comment by phhusson** [comment]
Yup as expected. I already am debugging those issues with someone else on
Telegram at t.me/phhtreble

**Comment by drobec255** [comment]
Hello, i wanted to flash LineageOS 17.1 to Motorola One Macro, but now i am frozen at booting. Phone is permanently restarting... pls help
